packages/rn-tester/RNTester/AppDelegate.h

```diff
@@ -5,9 +5,13 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#import <RCTAppDelegate.h>
+#import <RCTReactNativeFactory.h>
+#import <RCTDefaultReactNativeFactoryDelegate.h>
 #import <UIKit/UIKit.h>
 
-@interface AppDelegate : RCTAppDelegate
+@interface AppDelegate : RCTDefaultReactNativeFactoryDelegate <UIApplicationDelegate, UISceneDelegate>
+
+@property(nonatomic, strong, nonnull) UIWindow *window;
+@property(nonatomic, strong, nonnull) RCTReactNativeFactory *reactNativeFactory;
```

Comment:
maybe this should be `readonly`?

Reply:
I need to write to both of these properties in AppDelegate: 

![CleanShot 2025-02-03 at 15 04 38@2x](https://github.com/user-attachments/assets/5463f781-f718-4c09-b426-17fbbd35c045)

Same with window

Reply:
😭 We don't have a `private(set)` in objective-c, have we?

Reply:
Unfortunately not 😢 We could work around this by overwriting setters but it's user's AppDelegate not API from RN so Im not sure if its worth doing